libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccount.m

```diff
@@ -78,14 +78,6 @@ + (BOOL)supportsSecureCoding {
     return YES;
 }
 
-- (instancetype)init {
```

Comment:
Update: the only way I see to get rid of the proliferation of warnings around this area is either a) re-defining `init` with `NS_UNAVAILABLE`, or b) taking `NS_DESIGNATED_INITIALIZER` off of `initWithCredentials`.  Ugh.